java/ql/src/Security/CWE/CWE-940/AndroidIntentRedirection.qhelp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+    <overview>
+        <p>An exported Android component that obtains a user-provided Intent and uses it to launch another component
+        can be exploited to obtain access to private, unexported components of the same app or to launch other apps' components
+        in behalf of the victim app.</p>
+    </overview>
+    <recommendation>
+        <p>Do not export compontents that start other components from a user-provided Intent. 
+        They can be made private by setting the `android:exported` property to `false` in the app's Android Manifest.</p>
+        <p>If this is not possible, restrict either which apps can send Intents to the affected component, or which components can be started from it.</p>
+    </recommendation>
+    <example>
+        <p>The following snippet contains two examples.
+        In the first example, an arbitrary component can be started from the externally provided `forward_intent` Intent.
+        In the second example, the destination component of the Intent is first checked to make sure it is safe.</p>
+        <sample src="AndroidIntentRedirectionSample.java" />
+    </example>
+    <references>
+        <li>
+            Google:
+            <a href="https://support.google.com/faqs/answer/9267555?hl=en">Remediation for Intent Redirection Vulnerability</a>.
+        </li>
+        <li>
+            OWASP Mobile Security Testing Guide:
+            <a href="https://mobile-security.gitbook.io/mobile-security-testing-guide/android-testing-guide/0x05a-platform-overview#intents">Intents</a>.
+        </li>
+        <li>
+            Android Developers:
+            <a href="https://developer.android.com/guide/topics/manifest/activity-element#exported">The `android:exported` attribute</a>.
```

Comment:
Also here, but I don't know whether <code> works inside <a>.

Reply:
I tried, but it doesn't work. I just removed the backticks in 4bf22f2. Thanks!